src/UserActivity.ts

```diff
@@ -182,17 +182,20 @@ export default class UserActivity {
         this.activeRecentlyTimeout.abort();
     };
 
-    private onUserActivity = (event: MouseEvent): void => {
+    private onUserActivity = (event: Event): void => {
         // ignore anything if the window isn't focused
         if (!this.document.hasFocus()) return;
 
-        if (event.screenX && event.type === "mousemove") {
-            if (event.screenX === this.lastScreenX && event.screenY === this.lastScreenY) {
+        if (event.type === "mousemove" && (event as MouseEvent).screenX) {
```

Comment:
Can we use a custom type guard here instead? 

```
function isMouseEvent(event: Event): event is MouseEvent {
    return event.type.startsWith("mouse");
}
```

then using that before the `event.type` check to tell TS more about the type. Alternative an `isMouseMove` to be able to skip the event.type check

Reply:
Nice! TIL about type predicates